src/ResourceManager/Insights/Commands.Insights/OnetimeGenerated/Insights/MetricDefinitionOperations.cs

```diff
@@ -0,0 +1,505 @@
+// 
```

Comment:
Why are these files in a different folder than the rest of the geernerated client operations